### ISSUE TITLE
Buttons not centering

### ISSUE THREAD

**Issue by HardeepAsrani** [Codeinwp/gutenberg-blocks#614]
Few users reported that they haven't been able to center the buttons since the 1.6 update.

**Comment by mghenciu** [comment]
Related: https://github.com/Codeinwp/neve/issues/2655
https://wordpress.org/support/topic/multiple-button-alignment/

**Comment by HardeepAsrani** [comment]
This was fixed in last night's release. @mghenciu - Is it still happening?

**Comment by mghenciu** [comment]
Imported this 2 demos, that had alignment issues before:
- https://demosites.io/laundry-services-gb
- https://demosites.io/rokophoto-gb/

**and the alignment it's ok after import.**

Thank you, @HardeepAsrani !

**Comment by mghenciu** [comment]
@HardeepAsrani 
Just updated Otter on demosites.io to 1.6.3, and YES unfortunately the issue with button-group buttons alignment is back.
Basically it's what the customer mentioned in the wp.org ticket above, after you update Otter the buttons 'go crazy'.

A few examples:
![image](https://user-images.githubusercontent.com/52494172/113564224-a6ae8e00-9611-11eb-9c5b-b54020866b31.png)
![image](https://user-images.githubusercontent.com/52494172/113564370-f4c39180-9611-11eb-88ad-d1d993cb5f8d.png)
![image](https://user-images.githubusercontent.com/52494172/113564426-0a38bb80-9612-11eb-9777-f4e86d88f260.png)

**I rolled back to version 1.6.1**, until things clear up.

**Comment by HardeepAsrani** [comment]
@mghenciu Still works for me. Can you put it on a test site and share the creds with me?

**Comment by mghenciu** [comment]
I've tried to replicate on a test instance, but didn't managed to 'obtain a successful result'.
**tl;dr:** Yes, following the Test Scenario, meaning to have a site with Otter 1.6.1 and then update to 1.6.3 -> fixes the issue, but that's not what **happens on demosites.io, and it may happen/happened to some clients.**

You can try tomorrow (for a short time) to update Otter to 1.6.3 on demosites.io, and check one of this demos to get more technical details:
- https://demosites.io/laundry-services-gb/
- https://demosites.io/photography-gb/


A fresh jurassic with: Neve + **Otter 1.6.1** & Photography demo imported
https://resonant-eagle.jurassic.ninja 
User: demo yDkenQ7y3pXr 
![image](https://user-images.githubusercontent.com/52494172/113628731-fe7ae280-966d-11eb-8447-49e6bd260189.png)

Jurassic with: Neve + **Otter 1.6.3** & Laundry services demo imported
https://detailed-scarlet.jurassic.ninja 
User: demo I9Rsg0qIWp6Z 
Apparently here, it's working as expected after **Import**
![image](https://user-images.githubusercontent.com/52494172/113629491-24ed4d80-966f-11eb-823f-a09ecedbef68.png)

**Comment by HardeepAsrani** [comment]
@mghenciu I've released a new version, can you see if this works after updating?

**Comment by mghenciu** [comment]
@HardeepAsrani 
Alignment looks good.
Thank you!

Tested on [3 demos mentioned above](https://github.com/Codeinwp/gutenberg-blocks/issues/614#issuecomment-809371706), and also imported two of them on Jurassic, and the alignment works as expected.
